public/app/features/logs/components/panel/LogLineDetails.tsx

```diff
@@ -104,15 +59,15 @@ const getStyles = (theme: GrafanaTheme2) => ({
   container: css({
     overflow: 'auto',
     height: '100%',
+    boxShadow: theme.shadows.z1,
+    border: `1px solid ${theme.colors.border.medium}`,
+    borderRight: 'none',
   }),
   scrollContainer: css({
     overflow: 'auto',
-    position: 'relative',
     height: '100%',
   }),
-  closeIcon: css({
-    position: 'absolute',
-    top: theme.spacing(1),
-    right: theme.spacing(1.5),
+  componentWrapper: css({
+    padding: theme.spacing(0, 1, 1, 1),
```

Comment:
oh god it takes multiple args? I've been doing padding: `${theme.spacing(1)}, ${theme.spacing(1)}, ${theme.spacing(1)}, ${theme.spacing(1)}` jeeeez